File: 프로그래머스/lv1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.js
function solution(s) {
    var answer = '';
    if (s.length % 2 == 1){
        answer = s[parseInt(s.length / 2)];
    }
    else {
        answer = s.split("").slice(parseInt(s.length/2)-1, parseInt(s.length/2)+1).join("");
    }
    return answer;
}

File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        int idx = (int)s.length() / 2;
        if (s.length() % 2 == 0) {
            answer = String.valueOf(s.charAt(idx-1)) + String.valueOf(s.charAt(idx));
        } else {
            answer = String.valueOf(s.charAt(idx));
        }
        return answer;
    }
}